_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused `mouseover` events to focus buttons and prevent keyboard navigation. 
+    summaryAdditional:  When you hover your mouse over the date picker buttons, only the hover state will be triggered.
```

Comment:
**suggestion:** Some small tweaks for clarity. 

```suggestion
    summary: Fixed a bug that caused `mouseover` events to prevent keyboard navigation. 
    summaryAdditional:  Now when you hover your mouse over the date picker buttons, only the hover state will be triggered.
```

Reply:
Sounds good! Resolved in [9d378a4](https://github.com/uswds/uswds-site/pull/2496/commits/9d378a4d5bfb51b006f5c2d48bfbf0369b57b859)